cmk/base/plugins/agent_based/esx_vsphere_hostsystem_section.py

```diff
@@ -34,7 +34,21 @@ def parse_esx_vsphere_hostsystem(string_table: StringTable) -> Section:
     return section
 
 
+def host_label_function(section: Section) -> HostLabelGenerator:
+    """Host label function
+
+    Labels:
+
+        cmk/vsphere_object:
+            This label is set to "vcenter" if the corresponding host is a
+            VMWare vCenter, and to "server" if the host is an ESXi hostsystem.
+
+    """
+    yield HostLabel("cmk/vsphere_object", "server")
```

Comment:
Please change the docstring to reflect the behavior of the function. Ideally use a sentence why it makes sense that this only ever returns this label.